### PR TITLE
Add small files to tower.yml

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
Add output files information to the file `tower.yml`. 

Close https://github.com/nf-core/taxprofiler/issues/373
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tower.yml [hunk 1]
```diff
@@ -1,5 +1,135 @@
 reports:
   multiqc_report.html:
     display: "MultiQC HTML report"
-  samplesheet.csv:
```
Comment (LilyAnderssonLee) [inline tower.yml:4]: @jfy133 I guess I should keep the `samplesheet.csv` and add `database.csv` info here, right?

Comment (LilyAnderssonLee) [reply tower.yml:4]: No, `samplesheet.csv` is the input.

**Comment by LilyAnderssonLee** [comment]
> @LilyAnderssonLee I think you were a bit overboard.
> 
> The idea is to be able to visualize quickly text file and html reports (cf https://github.com/nf-core/rnaseq/blob/dev/tower.yml). I would remove zip and gzip files.
> 
> cf https://docs.seqera.io/platform/23.4.0/reports/overview#limitations

Ok, I will update it.

File: tower.yml (hunk 1 above)
Comment (maxulysse) [reply tower.yml:4]: then yeah, that's exactly what you'd want to do

**Review by JoseEspinosa — APPROVED** [review]
LGTM

**Review by JoseEspinosa — COMMENTED** [review]
Sorry did not see @maxulysse review, please address his suggestions and fix failing tests before merging

**Comment by jfy133** [comment]
@LilyAnderssonLee do you know how fancythe TSV viewer is? 

If you can do filtering of tables indeed makes sense to include the taxonomic profiles, but if not I'm not sure the utility of inmcluding those.

HTML reports makes sense, asn does maybe log files, but otherwise...?

**Comment by LilyAnderssonLee** [comment]
@jfy133 TBH I am not sure about the TSV viewer. I have never run taxprofiler on tower yet. It seems tower reports only support these formats: HTML, csv, tsv, pdf, and txt.

**Comment by jfy133** [comment]
Apparently it does! (using nf-core/rnaseq on the AWS megatest workspace as an example!)

image

**Comment by jfy133** [comment]
I have a feeling some of the tables will fail to render properly as extension is `.txt` but Mimetype is not set to tSV

But I think we can review this and fix it in the future release, once we have a fulltest run to inspect the output files :)

**Review by jfy133 — APPROVED** [review]

**Pull request merged**